.github/workflows/ci.yml

```diff
@@ -20,8 +20,11 @@ jobs:
   build-linux:
     runs-on: ubuntu-latest
     steps:
+    - name: Update apt sources
+      run: sudo apt-get update
+
     - name: Install Qt
-      uses: jurplel/install-qt-action@v2
+      run: sudo apt-get install qtbase5-dev qttools5-dev qttools5-dev-tools
```

Comment:
Thank you! This was a big help in solving. Couldn't figure out which dependency was missing:
~~~
In file included from Constellation.h:28,
                 from Constellation.cpp:21:
Constellation.cpp: In member function ‘void Constellation::drawAxes()’:
Constellation.cpp:101:32: error: ‘I’ was not declared in this scope; did you mean ‘PI’?
  101 |     SUCOMPLEX delta = SU_C_EXP(I * angle);
      |                                ^
make[1]: *** [Makefile.Release:534: release/Constellation.o] Error 1
make[1]: Leaving directory '/home/stevecrozz/dev/SuWidgets'
make: *** [Makefile:42: release] Error 2
~~~